src/storage/v3/expr.cpp

```diff
@@ -0,0 +1,216 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include "storage/v3/expr.hpp"
```

Comment:
General remark on the file: I think functions here have been taken from shard_rsm.cpp (see https://github.com/memgraph/memgraph/pull/578).
Once the other PR is merged, I think one should double check no extra modifications were done in shard_rsm.cpp.
Also, functions in shard_rsm.cpp should be removed.